packages/clerk-js/src/core/clerk.ts

```diff
@@ -1009,6 +1010,11 @@ export class Clerk implements ClerkInterface {
       newSession = this.#getSessionFromClient(newSession?.id);
     }
 
+    if (newSession?.status === 'pending') {
```

Comment:
This handles the case where a session remains `pending` after `/touch`, eg: a next task after selecting an organization